src/AppBundle/Entity/User.php

```diff
@@ -78,9 +78,9 @@ class User extends BaseUser
     /**
      * @var bool
      *
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
I did and it throws and exception in the database level. Do you really test this in your end?